storage-service/src/minio.ts

```diff
@@ -100,7 +101,7 @@ const upload = (
     file,
     s,
     metaDataWithName,
-    (err: any, etag: any) => {
+    (err: unknown, etag: unknown) => {
```

Comment:
do not use `unknow` as type. more information [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/40f00cd698640f1d8077f4d52eed1b5d2b2a7838/types/minio/index.d.ts)